### PR TITLE
Added support for hourly chores

### PR DESCRIPTION
**Pull request by andreheuer**
## Description
I have added a new line to support the new period type "hourly".

**Related issue (if applicable):** fixes #custom-components/grocy/issues/205

## Checklist
  - [x] The code change is tested and works locally.
  - [x] tests pass. **Your PR won't be merged unless tests pass**
  - [x] There is no commented out code in this PR

### CODE REVIEW

**Review by SebRut — DISMISSED** [review]

**Comment by andreheuer** [comment]
@SebRut Could you review (unfortunately again) the PR and merge it? I would like to open another PR for the test update ;-)

**Review by SebRut — APPROVED** [review]

**Comment by SebRut** [comment]
@andreheuer  LGTM! Out of curiosity, can you merge the PR by yourself after I approved it?

**Comment by andreheuer** [comment]
Nope, I cannot approve it. I just can close the PR, but then it is not merged (thats what I unintentionally before). So, you have to merge it.

**Pull request merged**
